frontend/common/src/components/tabs/TabSet.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from "react";
+import { TabProps } from "./Tab";
+
+export interface TabSetProps extends React.HTMLProps<HTMLElement> {
+  children: React.ReactElement<TabProps>[];
+}
+
+interface TabSetState {
+  selectedTab: number;
+  isOpen: boolean;
+}
+
+const firstSelectableTab = (tabs: React.ReactElement<TabProps>[]): number => {
+  const firstIndex = tabs.findIndex(
+    (tab: React.ReactElement<TabProps>) =>
+      tab.props.variant == null || tab.props.variant === "normal",
```

Comment:
Did you mean to for undefined instead of null? I don't think variant can ever be null, at least according to typescript.

Reply:
Good catch! 👍